katalog/node-exporter/sm.yml

```diff
@@ -7,9 +7,19 @@ metadata:
   namespace: monitoring
 spec:
   endpoints:
-  - interval: 30s
-    port: http
-    scheme: http
+  - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+    interval: 15s
```

Comment:
The interval was reduced from 30 -> 15s. Is intended? how does it affect?

Reply:
It affects our installations by doubling (?) the storage needed for node_exporter's metrics, right @valecontenti?

We should check the dashboards relaying on metrics from node_exported if there are any `rate()` queries (or, in general, PromQL queries ranging over a time interval) that rely on this scraping interval.

Ref: https://www.robustperception.io/what-range-should-i-use-with-rate

Reply:
Robust perception says: "The general rule for choosing the range is that it should be at least 4x the scrape interval. This is to allow for various races, and to be resilient to a failed scrape"

Both new and old alerts/rules/dashboards use a [1m] interval on rate() for node exporter metrics - the new 15s interval respects the above-mentioned 4x rule of thumb.
However is not a strict requirement, IMO, and if we're concerned about storage need we can revert back to 30s.

Reply:
I think we should stick to 15s because in the past we had some issues about gaps in Grafana graphs due to this.